README.rst

```diff
@@ -85,6 +85,15 @@ Globally (homebrew)
 
     $ brew install php-cs-fixer
 
+Locally (PHIVE)
+~~~~~~~~~~~~~~~~~~~
+
+Install `PHIVE <https://phar.io>`_ and issue the following command:
+
+.. code-block:: bash
+
+    $ phive install php-cs-fixer # use `--global` for global install, not recommended
```

Comment:
I would drop `, not recommended`. It's opinionated, and even while I agree with it, we don't put that for other ways of installation, don't we ?
Here, i brings the confusion that installing sth with PHIVE globally is not recommended, it's not true.
We simply don't recommend to install dev-tools globally (as their versions depends on project), regardless which way of installation was chosen

Reply:
i would keep it

Reply:
i would keep it